justfile

```diff
@@ -32,12 +32,21 @@ install-doppelganger:
         --bin doppelganger-parachain \
         --bin polkadot-execute-worker \
         --bin polkadot-prepare-worker  \
-        --locked --root ${DOPPELGANGER_PATH}
+        --locked --force --root ${DOPPELGANGER_PATH}
 
 # Install the `zombie-bite` binary on your system.
 install-zombie-bite:
     cargo install --git https://github.com/pepoviola/zombie-bite --bin zombie-bite --locked --force
 
+# Install the AHM Monitor
+install-monitor:
+    mkdir -p ./ahm-monitor/backend/data
+    cd ahm-monitor/backend \
+    && npm install \
```

Comment:
`&&` instead of `#!/usr/bin/env bash` + `set -e` for brevity or why?

Reply:
`&&` really mainly for staying in same dir, `just` runs each line of a recipe with working directory as the root. I.e. here `cd` changes directory from the root to `ahm-monitor/backend`, then the next line is run in the root:
```justfile
test:
    mkdir tmp
    cd tmp
    touch test
```
you'd end up with the behaviour of `touch ./test`, rather than the expected `touch ./tmp/test`

You can fix it with a shebang so it runs it all in the same instance, but I slightly prefer this as the recipes are then consistent, rather than throwing a shebang into some recipes but not others. don't feel too strongly about it though, happy to change if you think it's better style